config/jobs/kubernetes/sig-release/conformance-gate.yaml

```diff
@@ -0,0 +1,14 @@
+periodics:
+- interval: 4h
+  name: conformance-gate
+  decorate: true
+  spec:
+    containers:
+    - name: apisnoop-gate
+      image: zachboyofdestiny/snoopdb:gate
+      ports:
+      - name: postgres
+        containerPort: 5432
+      env:
+      - name: K8S_JOB
```

Comment:
FWIW there are a number of standard environment variables present in kubernetes CI
https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#job-environment-variables

Reply:
Good point, @BenTheElder!  I updated the underlying image to use a standard env var, and removed my redundant one.  This is part of commit a0c307c431521046333762492c402edab4e00c61